geomesa-gs-plugin/geomesa-kafka-gs-plugin/pom.xml

```diff
@@ -24,7 +24,7 @@
     <dependencies>
         <dependency>
             <groupId>org.locationtech.geomesa</groupId>
-            <artifactId>geomesa-kafka-datastore</artifactId>
+            <artifactId>geomesa-kafka-10-datastore</artifactId>
```

Comment:
Probably dovetails into @jahhulbert-ccri 's changes to the distribution structure.